module/scripts/index/import-from-local-dir-form.html

```diff
@@ -1,16 +1,17 @@
 <form bind="form">
-    <div class="grid-layout layout-normal">
-        <table role="presentation" style="width:80%;">
-            <tr>
-                <td colspan="2" id="or-import-enterdir"></td>
-            </tr>
-            <tr bind="urlRow">
-                <td colspan="2" style="white-space: nowrap"><input bind="dirInput" name="directory" type="text" class="default-importing-web-url" /></td>
-            </tr>
-            <tr bind="buttons">
-                <td width="1"><button bind="addButton" class="button" type="button"></button></td>
-                <td><button type="submit" bind="nextButton" class="button button-primary" type="button"></button></td>
-            </tr>
-        </table>
+  <div class="grid-layout layout-normal">
+    <div class="drive-selector-container">
+      <label id="driveSelectorLabel">Select drive / folder</label>
```

Comment:
There are a couple of strings in this file that would be worth localizing.

Reply:
@wetneb Sorry I missed this. will update